src/main/java/frc/robot/subsystems/IntakeFourBar.java

```diff
@@ -0,0 +1,37 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.Constants.RobotMap;
+
```

Comment:
Is there a reason for having 2 seperate subsystems? If so, maybe include a comment/Javadocs explaining why so that we stay consistent.

Reply:
Fixed the unnecessary import !

Reply:
Great! I'm asking more about the entire class tho... we have an `IntakeSubsystem` class, so why did we choose to not include the four-bar control logic in that class?

Reply:
Because we might want to do them in separate commands and there are shenanigans when you want two commands to use the same subsystem, according to @tervay

Reply:
@unbun We split intake into 2 different subsystems to avoid command requirements issues when we want to both deploy and run the rollers at the same time. i did this on 5254 in 2019 and it worked well. the alternative is to simply be more attentive to calling `addRequirements` in each command, but that seems like we are asking for a bug.

Reply:
makes sense, i was just thinking that if we put a sensor on the intake, i'm not sure what subsystem that would go on, but we can cross that bridge later